app/core/main_fastapi.py

```
@@ -0,0 +1,424 @@
+                "language_code": last_lang2,
+                "query": result.query,
+                "answer": result.answer,
+                "trace_id": trace_id,
+            }, ensure_ascii=False))
+
+            await websocket.send_text(json.dumps({"type": "done", "trace_id": trace_id}))
+            await websocket.close()
+
+    except WebSocketDisconnect:
+        return
+    except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)